man/systemd-boot.xml

```diff
@@ -146,7 +146,7 @@
       <varlistentry>
         <term><keycap>-</keycap></term>
         <term><keycap>T</keycap></term>
-        <listitem><para>Decrease the timeout</para></listitem>
+        <listitem><para>Decrease the timeout (decrease to -1 to unset custom timeout)</para></listitem>
```

Comment:
That doesn't sound good either. Things should be monotonic. I.e. if pressing "-" five times makes the timeout lower each time, pressing it for the sixth time shouldn't suddenly make it infinitely bigger.

Reply:
Note that `decrease to -1 to unset custom timeout` is already the current sd-boot behavior (without this change). This clears the custom timeout setting in the `LoaderConfigTimeout` EFI variable. All I did here is actually documenting this existing behavior.

Reply:
maybe add a new hotkey for turning it off altogether

Reply:
Adding a new hotkey also has been discussed in this [comment](https://github.com/systemd/systemd/pull/16320#issuecomment-652333110).